### ISSUE TITLE
Artifact in Steam with Nvidia 550.90 and 555.

### ISSUE THREAD

**Issue by reydeljuego12345** [ValveSoftware/gamescope#1383]
No matter what driver I install, I have always the same problem with artifact in the page of the store, flickering in menus, and black horizontals gaps on the menus. I'm tested with the new nvidia drivers in beta and with the old ones, and the problem persist. I'm using i3wm with picom as compositor.

**Comment by matte-schwartz** [comment]
this is not a gamescope issue: https://github.com/ValveSoftware/steam-for-linux/issues/10537#issuecomment-2170630863

**Comment by reydeljuego12345** [comment]
> this is not a gamescope issue: [ValveSoftware/steam-for-linux#10537 (comment)](https://github.com/ValveSoftware/steam-for-linux/issues/10537#issuecomment-2170630863)

Yes, You are right, when GPU accelerated hardware are not enable, all work well, so I guess this is a Chromium problem. But I notice that the artifact with the mouse cursor is not the same issue, because I have the same problem with other games. For example with lutris games. 

![screen](https://github.com/ValveSoftware/gamescope/assets/171271511/13f8d64f-04c9-4741-82a9-a53a053567d0)

![screen2](https://github.com/ValveSoftware/gamescope/assets/171271511/fd502582-ff4f-4251-bb37-21eb694f2540)

As you can see, the cursor is not complete either in the menu of those games. 

Is seem that this issue only happens with 4K resolution in some games, but in others, no matter the resolution.

**Comment by Gibitwit** [comment]
I have also had gamescope cut cursors short.  Shadow Warrior 2013 (Linux version) also has this problem.  Seems to me Gamescope has a cursor size limitation.

**Comment by reydeljuego12345** [comment]
> I have also had gamescope cut cursors short. Shadow Warrior 2013 (Linux version) also has this problem. Seems to me Gamescope has a cursor size limitation.
Are you in Nvidia too?

**Comment by Gibitwit** [comment]
Yes, Nvidia GPU here.

**Comment by Gibitwit** [comment]
Still occurs with Gamescope 3.14.24-11 and Nvidia drivers 560.31.02.

**Comment by reydeljuego12345** [comment]
> Still occurs with Gamescope 3.14.24-11 and Nvidia drivers 560.31.02.

I tried changing the size of the cursor but I cannot understand well how to reduce the size to prove if the problem is the size. Maybe chancing de default size in all the system

**Comment by MNarath1** [comment]
> > Still occurs with Gamescope 3.14.24-11 and Nvidia drivers 560.31.02.
> 
> I tried changing the size of the cursor but I cannot understand well how to reduce the size to prove if the problem is the size. Maybe chancing de default size in all the system

funny enough increasing the cursor size has given me some interesting results that make me question whats actually going on here since while the base cursor now looks broken the pointer cursor is suddenly more complete

**Comment by matte-schwartz** [comment]
the cursor issue is a Nvidia driver problem already being tracked here: https://github.com/ValveSoftware/gamescope/issues/1099#issuecomment-2231898778
